crunch.opam

```diff
@@ -1,34 +1,52 @@
+# This file is generated by dune, edit dune-project instead
```

Comment:
I'm not super fond of generated opam files - (I don't like the explicit dependency on `odoc` for instance). Could you revert that commit?

Reply:
Dropped the commit.